tools/schemacode/bidsschematools/__init__.py

```diff
@@ -13,5 +13,9 @@
 ]
 
 version_file = files("bidsschematools.data") / "schema" / "SCHEMA_VERSION"
-with as_file(version_file) as path:
-    __version__ = path.read_text().strip()
+with as_file(version_file) as f:
+    __version__ = f.read_text().strip()
```

Comment:
I don't think `SCHEMA_VERSION` should be the version of the library , but that is separate issue from this PR

Reply:
yep, we have a dedicated issue for that